sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -273,6 +281,10 @@ def run_pipeline(self, pipeline, options):
         pipeline_options.DirectOptions).direct_runner_bundle_repeat
     self._profiler_factory = profiler.Profile.factory_from_options(
         options.view_as(pipeline_options.ProfilingOptions))
+
+    if experiments and 'use_sdf_bounded_source' in experiments:
```

Comment:
```suggestion
    if 'use_sdf_bounded_source' in experiments:
```

You already guarantee that experiments is not `None` when creating the variable